frontend/javascripts/oxalis/model/bucket_data_handling/pullqueue.ts

```diff
@@ -31,6 +31,7 @@ class PullQueue {
   private abortController: AbortController;
   private consecutiveErrorCount: number;
   private isRetryScheduled: boolean;
+  private isDestroyed: boolean = false;
```

Comment:
why is this needed now for the tests to work? While checking the changes in the test file I did not notice anything obvious why this might be needed now :thinking:

Reply:
I just found something in `apiHelpers`. But this doesn't explain to me why this is now needed for testing and wasn't required previously